### PR TITLE
Updated Readme to include support to OCPP 2.0.1 

### LINKED ISSUE #143: Update README to mention OCPP 2.0.1 support.

### PR DESCRIPTION
**Pull request by tropxy**
Enhanced the example by checking the subprotocols supported by the CS

fixes #143

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -54,68 +55,81 @@ code in the `Central System documentation_`.
...
+    from ocpp.routing import on
+    from ocpp.v201 import ChargePoint as cp
+    from ocpp.v201 import call_result
+
+    logging.basicConfig(level=logging.INFO)
+
+
+    class ChargePoint(cp):
+    @on('BootNotification')
+    def on_boot_notitication(self, charging_station, reason, **kwargs):
```
Comment (laysauchoa) [inline README.rst:72]: missing the 4 spaces ;)

Comment (tropxy) [reply README.rst]: Damn you fast Laysa haha

File: README.rst [hunk 2]
```diff
@@ -54,68 +55,80 @@ code in the `Central System documentation_`.
...
+                interval=10,
+                status='Accepted'
+            )
+
+
+    async def on_connect(websocket, path):
+        """ For every new charge point that connects, create a ChargePoint
+        instance and start listening for messages.
+        """
+        requested_protocols = websocket.request_headers['Sec-WebSocket-Protocol']
```
Comment (OrangeTux) [inline README.rst:84]: The bounds of `websocket.request_headers` are not checked. This code will break with a `KeyError` if a client doesn't send the 'Sec-WebSocket-Protocol' header.

**Review by OrangeTux — CHANGES_REQUESTED** [review]
Can you also update the `License` section and mention that  OCPP 2.0.1 are not MIT licensed?

**Review by OrangeTux — CHANGES_REQUESTED** [review]
This comment hasn't been resolved yet:

> Can you also update the License section and mention that OCPP 2.0.1 are not MIT licensed?

**Review by OrangeTux — APPROVED** [review]

**Pull request merged**